cmd/init.go

```diff
@@ -22,10 +22,37 @@ func initCmd() *cobra.Command {
 			common.PanicIfError(err)
 			defaultConfig.OperatorAddress = address.String()
 			config.WriteConfigFile("./config.toml", &defaultConfig)
-			gen := config.DefaultGenesis()
-			if err := config.WriteGenesisFile(gen); err != nil {
-				panic(err)
-			}
+		},
+	}
+}
+
+func configureGenesisCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "load",
+		Short: "Load Genesis from a genesis.json",
+		Run: func(cmd *cobra.Command, args []string) {
+			cfg, err := config.ParseConfig()
+			common.PanicIfError(err)
+			genesis, err := config.ReadGenesisFile(args[0])
```

Comment:
Do you think we should use the flags here? Like the other cli commands?

Reply:
I was thinking about using the flag too. But then I realized the path is already a required argument and is the only argument. So we don't have to key in a flag in the command.